File: www(old)/eval/cgi-bin/hello.py
#!/usr/bin/env python3
import os
import sys

print("Content-Type: text/html\r\n\r\n")
print("<html><head><title>Python CGI</title></head><body>")
print("<h1>Hello from Python CGI! 🐍</h1>")
print("<h2>Environment Variables:</h2>")
print("<ul>")
for key, value in os.environ.items():
    print(f"<li><strong>{key}</strong>: {value}</li>")
print("</ul>")

if os.environ.get("REQUEST_METHOD") == "POST":
    body = sys.stdin.read()
    print("<h2>POST Body Received:</h2>")
    print(f"<pre>{body}</pre>")

print("</body></html>")

File: www(old)/eval/cgi-bin/hello.php
#!/usr/bin/env php-cgi
<?php
echo "Content-Type: text/html\r\n\r\n";
echo "<html><head><title>PHP CGI</title></head><body>";
echo "<h1>Hello from PHP CGI! 🐘</h1>";
echo "<h2>Environment Variables:</h2>";
echo "<ul>";
foreach ($_SERVER as $key => $value) {
    echo "<li><strong>$key</strong>: $value</li>";
}
echo "</ul>";

if ($_SERVER["REQUEST_METHOD"] == "POST") {
    $body = file_get_contents("php://input");
    echo "<h2>POST Body Received:</h2>";
    echo "<pre>$body</pre>";
}
echo "</body></html>";
?>
